lib/dock.rb

```diff
@@ -0,0 +1,34 @@
+class Dock
+  attr_reader :name, :max_rental_time, :rental_time
+  def initialize(name, max_rental_time)
+    @name = name
+    @max_rental_time = max_rental_time
+    @revenue_multiplier = []
+    @rental_time
+  end
+
+  def rent(boat, renter)
+    @revenue_multiplier << boat.price_per_hour
+    @rental_time = 0
+  end
+
+  def log_hour
```

Comment:
Per the spec, this method should add hours to the instances of Boats that have been rented from the Dock.